game/static/game/css/board.css

```
@@ -249,62 +371,142 @@ body {
+    justify-content: center;
+    background: rgba(8, 8, 16, 0.82);
+    border-radius: 2px;
+    pointer-events: none;
+}
+
+.countdown-overlay.active {
+    display: flex;
+}
+
+.countdown-number {
+    font-family: 'Cinzel', serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)